src/main/kotlin/app/revanced/patches/youtube/layout/theme/patch/CommentsFilterBarPatch.kt

```diff
@@ -0,0 +1,39 @@
+package app.revanced.patches.youtube.layout.theme.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.extensions.instruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.util.smali.ExternalLabel
+import app.revanced.patches.youtube.layout.theme.annotations.ThemeCompatibility
+import app.revanced.patches.youtube.layout.theme.fingerprints.CommentsFilterBarFingerprint
+
+@Name("comment-filter-bar-theme")
+@Description("Apply theme to comments filter action bar.")
+@ThemeCompatibility
+@Version("0.0.1")
+class CommentsFilterBarPatch : BytecodePatch(
+    listOf(
+        CommentsFilterBarFingerprint
+    )
+) {
+    override fun execute(context: BytecodeContext): PatchResult {
+        val result = CommentsFilterBarFingerprint.result!!
+        val method = result.mutableMethod
+        val patchIndex = result.scanResult.patternScanResult!!.endIndex - 1
+
+        method.addInstructions(
+            patchIndex, """
+                const v1, -0xdededf
```

Comment:
What is this magic constant? Can you explain it in a code comment?

Reply:
The constant is based on a color value (in this case, a tint of gray) from register p1, returned by a method which manages a bytebuffer (another one, not the same used by ads blocking).

The aformentioned method return also different other color values (for this reason I added a if-condition), with which you can change the color of other elements of the bar (for example: the colors of the pressed buttons).